src/SecTester.Reporter/DefaultFormatter.cs

```diff
@@ -52,12 +49,20 @@ private static string GenerateTemplate(bool extraInfo, bool references)
 
     if (extraInfo)
     {
-      stringBuilder.Append(TemplateExtraDetails);
+      stringBuilder
+        .Append(NewLine)
+        .Append(TemplateExtraDetails)
+        .Append(NewLine)
+        .Append("{5}");
     }
 
     if (references)
     {
-      stringBuilder.Append(TemplateReferences);
+      stringBuilder
+        .Append(NewLine)
+        .Append(TemplateReferences)
+        .Append(NewLine)
+        .Append("{6}");
```

Comment:
You just need to remove trailing or leading, not in the middle https://github.com/NeuraLegion/sectester-net/pull/112#discussion_r1046847904